.gitignore

```diff
@@ -17,6 +17,7 @@ android/gradle.properties
 /app/app-www/js/*
 !/app/app-www/js/left-drawer.js
 !/app/app-www/js/config.js
+!/app/app-www/js/global.js
```

Comment:
I wonder if we should throw all our third party js files into a folder like... `app-www/js/vendor/` and we ignore that folder... thoughts anyone?